client-templates/bitbucket/accept.json.sample

```diff
@@ -0,0 +1,47 @@
+{
+  "private":
+  [
+    {
+      "//": "list the user's projects",
+      "method": "GET",
+      "path": "/projects",
+      "origin": "https://${BITBUCKET_USERNAME}:${BITBUCKET_PASSWORD}@${BITBUCKET_API}"
+    },
+    {
+      "//": "list the user's repos",
+      "method": "GET",
+      "path": "/repos",
+      "origin": "https://${BITBUCKET_USERNAME}:${BITBUCKET_PASSWORD}@${BITBUCKET_API}"
+     },
+    {
+      "//": "used to determine the full dependency tree",
+      "method": "GET",
+      "path": "/projects/:project/repos/:repo/browse/package.json",
```

Comment:
How about adding `*` in the path to allow nested files? Same for Gemfile, pom.xml